BlockchainSdk/Common/Blockchain.swift

```diff
@@ -1234,3 +1234,15 @@ extension Blockchain {
         }
     }
 }
+
+// MARK: - Parallel transactions
+public extension Blockchain {
+    var isParallelTransactionAllowed: Bool {
```

Comment:
```suggestion
    var isParallelTransactionsAllowed: Bool {
```
можно еще поиграться с неймингом в сторону concurrent, multiple и т.д. - но кажется и так норм

Reply:
Тут тогда надо вместо is делать are, т.к. их же несколько. 

Я когда делал решил оставить в единственном числе с помощью такой мысленной проверки: "Могу ли я параллельно с текущей транзакцией отправить еще одну". 

То есть типа
- Блокчейн, ты тут?
- Ага
- Еще **одну** параллельную транзакцию можно?
- Давай

Reply:
>Тут тогда надо вместо is делать are, т.к. их же несколько.

все время забываю про to be(

по неймингу вроде понятно, что происходит - поэтому и не предлагаю что-то менять

Reply:
Только тут момент, зачем в extension выносить? Пусть лежит рядом с такими же пропертями